src/control/content/containers/Home.js

```diff
@@ -26,32 +26,33 @@ const Home = ({
 
     /** UTILS */
     const elimanateNotFoundImages = (url) => {
-      const optimisedURL = url.replace('1080x720', '100x100'); 
       return new Promise((resolve) => {
-        if (url.includes("http")){
+        if (url.includes("http")) {
           const xhr = new XMLHttpRequest();
-          xhr.open("GET", optimisedURL);
+          xhr.open("GET", url);
           xhr.onerror = (error) => {
-            console.warn('provided URL is not a valid image', error);
+            console.warn('Provided URL is not a valid image', error);
             resolve({isValid: false, newURL: null});
-          }
+          };
           xhr.onload = () => {
             if (xhr.responseURL.includes('source-404') || xhr.status == 404) {
-              return resolve({isValid: false ,newURL: null});
+              resolve({isValid: false, newURL: null});
             } else {
-              return resolve({isValid: true, newURL: xhr.responseURL.replace('h=100', 'h=720').replace('w=100', 'w=1080') });
+              resolve({isValid: true, newURL: xhr.responseURL});
             }
           };
           xhr.send();
-        } else resolve(false);
-        });
+        } else {
+          resolve({isValid: false, newURL: null});
+        }
+      });
     };
     /** UTILS END */
 
     const seeder = new buildfire.components.aiStateSeeder({
       generateOptions: {
         userMessage: 'Provide me with images that relate to [Traveling]',
-        systemMessage: 'Use source.unsplash.com for image URLs, image should be 1080x720, use different image for each entry, URL should not have premium_photo or source.unsplash.com/random. Do not remove white spaces in the subject string.',
+        systemMessage: 'Use https://app.buildfire.com/api/stockImages/?topic=topic&imageType=medium for image URLs, don\'\t use "topic" word as a topic  , A maximum of 2 comma-separated topics can be used for each URL, make sure to generate 10 images',
```

Comment:
Use https://app.buildfire.com/api/stockImages/?topic={topic}&imageType=medium for image URLs, and use "random" as a default, A maximum of 2 comma-separated issues can be used for each URL, make sure to generate 10 images